.github/workflows/ruby_ci.yml

```diff
@@ -0,0 +1,89 @@
+name: Ruby CI
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags-ignore:
+      - '*.*'
+  pull_request:
+    branches:
+      - '*'
+jobs:
+  test:
+    name: Ruby ${{ matrix.ruby }} ${{ matrix.name_extra || '' }}
+    runs-on: ${{ matrix.os || 'ubuntu-latest' }}
+    strategy:
+      matrix:
+        ruby:
+          - 3.0.0-preview2
+          - 2.7
+          - 2.6
+          - 2.5
+          - 2.4
+          - 2.3
+          - 2.2
+          - 2.1.9
+          - ruby-head
+        include:
+          - ruby: 'jruby-9.2.13.0'
+            env:
```

Comment:
It appears GHA doesn't like `env` being specified here. Apparently it's only supported in the `step` which isn't very helpful for us...

Reply:
@JonRowe are you sure the GHA flow in `rspec-core` is working?